transmart-api-server/src/main/groovy/org/transmartproject/api/server/user/AccessLevels.groovy

```diff
@@ -84,7 +84,7 @@ class AccessLevels {
             return true
         }
         Map<String, PatientDataAccessLevel> studyToPatientDataAccessLevel = buildStudyToPatientDataAccessLevel(authorities)
-        return !studyToPatientDataAccessLevel.empty
```

Comment:
is this not working only on Map objects? There are other places where this is used, but the cases I found are on List, Set and String objects

Reply:
I'm not sure...

Reply:
```
groovy:000> [].empty
===> true
groovy:000> ['a'].empty
===> false
groovy:000> [:].empty
===> null
groovy:000> "".empty
===> true
groovy:000> ([] as Set).empty
===> true
```
Seems to only fail on maps indeed

Reply:
I guess it conflicts with the syntactic sugar of addressing elements: `dict.key === dict[key]`, so `dict.empty` means `dict['empty']`...